references/classification/train.py

```diff
@@ -440,6 +440,7 @@ def get_args_parser(add_help=True):
     parser.add_argument("--lr-min", default=0.0, type=float, help="minimum lr of lr schedule (default: 0.0)")
     parser.add_argument("--print-freq", default=10, type=int, help="print frequency")
     parser.add_argument("--output-dir", default=".", type=str, help="path to save outputs")
+    parser.add_argument("--output-freq", default=1, type=int, help="output frequency")
```

Comment:
```suggestion
    parser.add_argument("--checkpoint-freq", default=1, type=int, help="Create a checkpoint every checkpoint-freq epochs")
```

Reply:
I agree, that `--checkpoint-freq` makes more sense but I wanted to stay consistent with `--output-dir`, perhaps changing `output-dir` to `checkpoint-dir`?

Reply:
All right! Changed it now to checkpoint-freq as suggested.